src/game_display.cpp

```diff
@@ -647,18 +654,33 @@ std::vector<texture> game_display::get_reachmap_images(const map_location& loc)
 	std::vector<std::string> names;
 	const auto adjacent = get_adjacent_tiles(loc);
 
-	enum visibility { REACH = 0, CLEAR = 1 };
+	enum visibility { REACH = 0, ENEMY = 1, CLEAR = 2 };
 	std::array<visibility, 6> tiles;
 
 	const std::string* image_prefix_ = &game_config::reach_map_prefix;
 	DBG_DP << "Loaded image prefix: " << game_config::reach_map_prefix;
 
+	// As a failsafe, make sure the team is initialized with a valid value
+	team t = viewing_team();
+	// Get the reachmap-context team, the selected unit's team shall override the displayed unit's.
+	if(context().units().count(selectedHex_)) {
+		t = resources::gameboard->get_team(context().get_visible_unit(selectedHex_, viewing_team())->side());
+	} else if(context().get_visible_unit(displayedUnitHex_, viewing_team()) != nullptr){
+		t = resources::gameboard->get_team(context().get_visible_unit(displayedUnitHex_, viewing_team())->side());
+	} 
```

Comment:
Just noting that the CI check seems to be complaining about a trailing space here.